docs/modules/ROOT/pages/how-tos/install/cloudscale.adoc

```diff
@@ -0,0 +1,211 @@
+
+. Create a new Syn cluster: https://control.vshn.net/syn/lieutenantclusters
+
+. Configure input
++
+[source,console]
+----
+export CLOUDSCALE_TOKEN=<cloudscale-token> # From https://control.cloudscale.ch/user/api-tokens
+export CLUSTER_NAME=<cluster-name>
+export BASE_DOMAIN=ocp4-poc.appuio-beta.ch
+export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+----
+
+. Create S3 buckets
++
+[source,console]
+----
+# Use already exiting bucket user
+curl -H "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  https://api.cloudscale.ch/v1/objects-users | \
+  jq -e ".[] | select(.display_name == \"${CLUSTER_NAME}\")"
+
+# Or create a new one
+response=$(curl -H "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  -F display_name=${CLUSTER_NAME} \
+  https://api.cloudscale.ch/v1/objects-users)
+
+export AWS_ACCESS_KEY_ID=$(echo $response | jq -r .keys[0].access_key)
+export AWS_SECRET_ACCESS_KEY=$(echo $response | jq -r .keys[0].secret_key)
+
+mc config host add \
+  s3 https://objects.rma.cloudscale.ch ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY}
+
+mc mb --ignore-existing\
+  "s3/${CLUSTER_NAME}-bootstrap-ignition" \
+  "s3/${CLUSTER_NAME}-tf-state"
+
+# Ignore the error
+mc policy set download "s3/${CLUSTER_NAME}-bootstrap-ignition"
+----
+
+. Prepare `install-config.yaml`
++
+[source,console]
+----
+mkdir ${CLUSTER_NAME}
+
+cat > "${CLUSTER_NAME}/install-config.yaml" <<EOF
+apiVersion: v1
+metadata:
+  name: ${CLUSTER_NAME}
+baseDomain: ${BASE_DOMAIN}
+compute:
+  - name: worker
+    replicas: 3
+controlPlane:
+  name: master
+  replicas: 3
+networking:
+  clusterNetwork:
+    - cidr: 10.128.0.0/14
+      hostPrefix: 23
+  networkType: OpenShiftSDN
+  serviceNetwork:
+    - 172.30.0.0/16
+platform:
+  none: {}
+pullSecret: |
+  ${PULL_SECRET}
+sshKey: "$(cat ~/.ssh/id_ed25519.pub)"
+EOF
+
+----
+
+. Prepare install manifests and ignition config
++
+[source,console]
+----
+openshift-install --dir ${CLUSTER_NAME} \
+  create manifests
+
+openshift-install --dir ${CLUSTER_NAME} \
+  create ignition-configs
+
+mc cp ${CLUSTER_NAME}/*.ign "s3/${CLUSTER_NAME}-bootstrap-ignition"
+----
+
+. Provision infrastructure
++
+[source,console]
+----
+terraform init \
+  -backend-config "bucket=${CLUSTER_NAME}-tf-state"
+
+export TF_VAR_cluster_name=$CLUSTER_NAME
+
+terraform apply \
+  -target=cloudscale_floating_ip.api_vip \
+  -var api_eip=""
+
+terraform refresh
+
+export TF_VAR_api_eip="$(terraform output -json | jq -r .api_vip.value)"
+
+terraform apply \
+  -target=cloudscale_floating_ip.api_vip \
+  -target=cloudscale_server.bootstrap \
+  -target=cloudscale_server.master \
+  -var bootstrap_count=1
+----
+
+. Create the necessary DNS records
++
+[source,console]
+----
+terraform refresh
+
+terraform output -json | jq -r .dns_entries.value
+----
+
+. Wait for bootstrap to complete
++
+[source,console]
+----
+openshift-install --dir ${CLUSTER_NAME} \
+  wait-for bootstrap-complete
+----
+
+. Remove bootstrap node
```

Comment:
Can be omitted. The next step will run that target anyway.

Reply:
I wanted this to be a separate step to make sure the bootstrap node is gone before adding nodes.

Reply:
Fair.